bluehost.php

```diff
@@ -46,10 +46,12 @@
 	if ( $did_upgrade ) {
 		update_option( 'bluehost_plugin_version', BLUEHOST_PLUGIN_VERSION, true );
 	}
-
 }
 
 // Require files
+require dirname( __FILE__ ) . '/inc/class-access-token.php';
```

Comment:
Can we follow wordpress class/file naming standards or agree on a PSR standard?
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

Reply:
@0aveRyan I think we should definitely use namespaces and PSR-4 autoloading. I've made a note to add, but also think that makes the most sense in another PR where we also remove our PHP 5.2 support.